api/pkg/backend/service.go

```diff
@@ -572,6 +651,16 @@ func (s *APIService) UpdateTier(request *restful.Request, response *restful.Resp
 	}
 
 	ctx := common.InitCtxWithAuthInfo(request)
+	//Validation of tenantId of tier:
+	actx := request.Attribute(c.KContext).(*c.Context)
+	token := actx.AuthToken
+	tenantid := request.PathParameter("tenantId")
+	if !ValidateTenant(token, tenantid) {
```

Comment:
the same code has been used at multiple place, Can we optimize something here??
***********
if !ValidateTenant(token, tenantid) {
		errMsg := fmt.Sprintf("tenantId:%v in request is invalid:", tenantid)
		log.Error(errMsg)
		response.WriteError(http.StatusBadRequest, errors.New(errMsg))
		return
	}*******

Reply:
that is an error msg